collections-csharp/MergingDictionariesInCSharp/MergingDictionariesInCSharp.Benchmark/MergingMethodsBenchmark.cs

```diff
@@ -0,0 +1,94 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Order;
+using static MergingDictionariesInCSharp.Methods;
+
+
+[MemoryDiagnoser]
+[Orderer(SummaryOrderPolicy.FastestToSlowest)]
+[RankColumn]
+public class MergingMethodsBenchmark
+{
+    private Dictionary<int, string> _dictionaryA;
+    private Dictionary<int, string> _dictionaryB;
+
+    [GlobalSetup]
+    public void Setup()
+    {
+        _dictionaryA = new Dictionary<int, string>();
+        _dictionaryB = new Dictionary<int, string>();
+
+        var random = Random.Shared;
+
+        // Generate random values for dictionaryA
+        for (var i = 0; i < 100; i++)
+        {
+            var key = random.Next(100);
```

Comment:
Now that I think about it, is there any point in generating random keys if we are simply iterating from 0-99. Moreover, if we find a key that already exists, we are decrementing `i` so we will always end up with 100 entries. It would simply be easier to add `i` as the key and generate a random value. Same for `_dictionaryB`, but 100-199.

Reply:
Ok, implemented it as described.